src/components/molecules/card/CardBackgroundImage.tsx

```diff
@@ -11,16 +11,18 @@ interface IProps {
   variant: HeaderVariant;
 }
 
-const useStyles = makeStyles({
+const useStyles = makeStyles<Theme, IProps>((theme) => ({
   img: {
     position: 'absolute',
     zIndex: -1,
     top: 0,
     right: 0,
+    marginInlineStart: (props) => (props.variant === HeaderVariant.Label ? `${theme.spacing(1)}px` : 0),
```

Comment:
I think there is no need for `${theme.spacing(1)}px`, you can do simply `theme.spacing(1)`
See the docs
https://material-ui.com/customization/spacing/#multiple-arity

Reply:
I saw that too, but it doesn't work for callbacks.
Turns out it's a known issue (https://github.com/mui-org/material-ui/pull/22552) that's already solved and merged to v5.